src/value/value.rs

```diff
@@ -116,30 +116,46 @@ impl Value {
     /// use figment::{value::Value, util::map};
     ///
     /// let value = Value::from(map! {
-    ///     "apple" => map! {
+    ///     "apple" => Value::from(map! {
     ///         "bat" => map! {
     ///             "pie" => 4usize,
     ///         },
     ///         "cake" => map! {
     ///             "pumpkin" => 10usize,
-    ///         }
-    ///     }
+    ///         },
+    ///     }),
+    ///     "duck" => Value::from(vec![
+    ///         map!{
+    ///             "pasta" => 42usize,
+    ///         },
+    ///         map!{
+    ///             "pizza" => 229usize,
+    ///         },
+    ///     ]),
```

Comment:
It would involve either the array being fairly deeply nested and/or having an array of mixed types. I instead opted to remove one value from each top-level key.